src/utils/use-news-handlers.js

```diff
@@ -1,6 +1,15 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
+import { FormControlLabel, Checkbox } from '@material-ui/core';
+import { config } from '../configs';
+
+const { languages } = config;
 
 const useNewsHandlers = () => {
+  const checkboxStates = languages.reduce(
+    (obj, lang) => ({ ...obj, [lang]: false }),
+    {}
+  );
+
   const [authorPhoto, setAuthorPhoto] = useState('');
   const [newsImage, setNewsImage] = useState('');
   const [newsVideo, setNewsVideo] = useState('');
```

Comment:
Do we still have `video` in article data?

Reply:
Removed!